### ISSUE TITLE
Remove deprecated twistd --nothotshot

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4237]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4237|
|Type|task|
|Created|2010-01-27 17:28:38Z|

The ```--nothotshot``` option of ```twistd``` is marked as deprecated so let's remove it.

<details><summary>Searchable metadata</summary>

```
trac-id__4237 4237
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__ 
time__1264613318000000 1264613318000000
changetime__1277152938000000 1277152938000000
version__None None
owner__ 
cc__thijs cc__cyli
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra set status to `closed`|
|-|-|

Seems this was fixed in r29008 ([#4091](https://github.com/twisted/twisted/issues/4091)) so will mark this as a duplicate since a fixed resolution would need a NEWS file I guess ;)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
